### ISSUE TITLE
Can't use Native MPP with gradle with 0.8-dev-2585

### ISSUE THREAD

**Issue by soywiz** [JetBrains/kotlin-native#1733]
I have created an issue in YouTrack too (since the fix could be required in the main kotlin repo): https://youtrack.jetbrains.com/issue/KT-25135

This configuration seems to work: `kotlinNativeVersion=0.8-dev-2494`, `kotlinVersion=1.2.41`

I can reproduce it with several Kotlin versions: Kotlin 1.2.41, Kotlin 1.2.50 and 1.2.60-eap-7

As far as I understand this is the culprit (so it is related to the Kotlin version kotlin-native includes):

https://github.com/JetBrains/kotlin/blob/46a3f7420ce200760359ef2a771bd634aef4ef09/libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/KotlinMultiplatformPlugin.kt#L216

When I bump versions from , in an `expectBy project()` I get this error:
* `kotlinNativeVersion=0.8-dev-2585`
* `Could not find any convention object of type JavaPluginConvention.`

If I apply the java plugin I get this error instead:
* `Cannot add task ':*:test' as a task with that name already exists.`

Stacktraces:
```
Caused by: java.lang.IllegalStateException: Could not find any convention object of type JavaPluginConvention.
        at org.gradle.api.internal.plugins.DefaultConvention.getPlugin(DefaultConvention.java:96)
        at org.jetbrains.kotlin.gradle.plugin.KotlinMultiplatformPluginKt.getSourceSets(KotlinMultiplatformPlugin.kt:216)
        at org.jetbrains.kotlin.gradle.plugin.KotlinMultiplatformPluginKt.access$getSourceSets$p(KotlinMultiplatformPlugin.kt:1)
        at org.jetbrains.kotlin.gradle.plugin.KotlinPlatformImplementationPluginBase.namedSourceSetsContainer(KotlinMultiplatformPlugin.kt:147)
        at org.jetbrains.kotlin.gradle.plugin.KotlinPlatformImplementationPluginBase$addCommonProject$1.invoke(KotlinMultiplatformPlugin.kt:113)
        at org.jetbrains.kotlin.gradle.plugin.KotlinPlatformImplementationPluginBase$addCommonProject$1.invoke(KotlinMultiplatformPlugin.kt:50)
        at org.jetbrains.kotlin.gradle.plugin.KotlinPlatformImplementationPluginBase$Companion.whenEvaluated(KotlinMultiplatformPlugin.kt:171)
```

and:

```
Caused by: org.gradle.api.InvalidUserDataException: Cannot add task ':dynarek-native:test' as a task with that name already exists.
        at org.gradle.api.internal.tasks.DefaultTaskContainer.addTask(DefaultTaskContainer.java:182)
        at org.gradle.api.internal.tasks.DefaultTaskContainer.create(DefaultTaskContainer.java:212)
        at org.gradle.api.internal.tasks.DefaultTaskContainer.create(DefaultTaskContainer.java:206)
        at org.gradle.api.internal.tasks.DefaultTaskContainer.create(DefaultTaskContainer.java:225)
        at org.gradle.nativeplatform.test.plugins.NativeTestingBasePlugin.apply(NativeTestingBasePlugin.java:56)
        at org.gradle.nativeplatform.test.plugins.NativeTestingBasePlugin.apply(NativeTestingBasePlugin.java:46)
```

**Comment by soywiz** [comment]
It seems that the applied `NativeTestingBasePlugin` is not compatible with the Java one. And the Java one is used by the KotlinMultiplatformPlugin for the sourceSets :)

https://github.com/JetBrains/kotlin-native/blob/9f64280bc7b5e8ebf729c4234e1750cede0c3f6f/tools/kotlin-native-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/experimental/plugins/KotlinNativePlugin.kt#L167

**Comment by soywiz** [comment]
Fixed in `0.8-dev-2598` #1721, as long as you don't include the Java plugin.
